domains/animesh-kotka.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Simple Portfolio",
+    "repo": "https://github.com/AnimeshKotka/Animesh-Kotka-Portfolio",
+    "owner": {
+        "username": "AnimeshKotka",
+        "email": "animesh@kotka.tech"
+    },
+    "record": {
+        "CNAME": "https://animeshkotka.github.io/Animesh-Kotka-Portfolio"
```

CNAME cannot contain protocol or `/`
```suggestion
        "CNAME": "animeshkotka.github.io"
```

---

done